src/components/Card/UserGroup/UserGroupCard.jsx

```diff
@@ -1,58 +1,30 @@
-import changeDayFormat from "../../../utils/changeDayFormat";
+import { Link } from "react-router-dom";
+
 import getDate from "../../../utils/getDate";
 import KeywordChip from "../../Chip/KeywordChip";
-import Button from "../../UI/Button";
 import PropTypes from "prop-types";
 
-const UserGroupCard = ({ groupId, groupName, keywordList, createdAt, updatedAt }) => {
-  const createdDate = getDate(createdAt);
+const UserGroupCard = ({ groupId, groupName, keywordList, updatedAt }) => {
   const updatedDate = getDate(updatedAt);
 
   return (
-    <div className="flex justify-between w-full border-3 border-rose-200/80 bg-white rounded-[30px] pl-30 pr-20 py-10 shadow-sm hover:border-rose-300/70">
-      <div className="flex flex-col items-start justify-center gap-3">
-        <p className="flex items-center gap-10">
-          <span className="text-purple-600 text-19 font-bold">그룹명: </span>
-          <span className="text-rose-400 text-18">{groupName}</span>
-        </p>
-        <div className="flex items-center gap-10 w-full">
-          <span className="text-purple-600 text-19 font-bold flex-shrink-0">키워드 리스트: </span>
-          <p className="flex items-center gap-5 w-full">
-            {keywordList.map((keyword) => (
-              <KeywordChip
-                key={keyword._id}
-                keywordName={keyword.keyword}
-                styles="flex-center text-14 px-5 py-2 bg-orange-100 text-rose-800 rounded-[3px]"
-              />
-            ))}
-          </p>
+    <Link to={`/dashboard/${groupId}`}>
+      <div className="flex flex-col items-start justify-center gap-7 w-full border-2 border-slate-200/80 bg-white rounded-[5px] px-30 py-12 shadow-sm hover:border-emerald-900/30 hover:shadow-md">
+        <p className="flex items-center tex-green-900 text-20 font-semibold">{groupName}</p>
+        <div className="flex items-center gap-5 w-full mb-2">
+          {keywordList.map((keyword) => (
+            <KeywordChip
+              key={keyword._id}
+              keywordName={keyword.keyword}
+              styles="flex-center text-14 px-5 py-2 bg-green-500/10 text-black rounded-[3px]"
+            />
+          ))}
         </div>
-        <p className="flex items-center gap-10">
-          <span className="text-purple-600 text-19 font-bold">그룹 생성일: </span>
-          <span className="text-rose-400 text-18">
-            {createdDate.currentYear}년 {createdDate.currentMonth}월 {createdDate.currentDate}일{" "}
-            {changeDayFormat(createdDate.currentDay)} {createdDate.currentHour}시{" "}
-            {createdDate.currentMinute}분
-          </span>
+        <p className="flex items-center gap-5 text-slate-500 text-13">
+          {updatedDate.currentYear}년 {updatedDate.currentMonth}월 {updatedDate.currentDate}일
```

Comment:
```suggestion
          {updatedDate.currentYear}년 {updatedDate.currentMonth}월 {updatedDate.currentDate}일 갱신됨
```

```suggestion
          {updatedDate.currentYear}년 {updatedDate.currentMonth}월 {updatedDate.currentDate}일 업데이트됨
```

아래 처럼 UI를 보면 해당 날짜가 어떤 의미의 날짜인지 인지가 안되더라구요. 
위 2개 예시처럼, 의미를 조금 살려도 좋겠다 생각했어요.

<img width="500" alt="Screenshot 2024-11-13 at 11 45 38 PM" src="https://github.com/user-attachments/assets/5cfb09a9-5f8b-44f0-8c2d-c8f7319cca05">

Reply:
아래의 아이콘을 날짜 문구 앞에 사용해봐도 좋겠다는 의견이네요 :)

[Update 아이콘](https://www.svgrepo.com/svg/502881/update)

Reply:
`(아이콘) 마지막 업데이트 : 2024년 11월 14일 `로 하는건 어떨까요?! 
사용자에겐 아이콘이 직관적이지 않을 수 있으니, 같이 쓰는 것도 좋을 것 같아요!

Reply:
"마지막 업데이트"라는 문구가 직관적이라 적용해봐도 좋겠습니다.

Reply:
아래와 같이 수정하였습니다 :)

![image](https://github.com/user-attachments/assets/3f4b9301-22ef-4e19-84bd-6f81fa4ab0fd)